app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_user!
+
+      include RentsControllerHelper
+
+      def index
+        render json: Rent.page(page).per(per_page), each_serializer: RentSerializer,
+               status: :ok
+      end
+
+      def create
+        return unless valid_user?(rent_params[:user_id]) && valid_book?(rent_params[:book_id])
```

Comment:
Is this ok? Any user can rent a book for any other user or the current_user rents a book for himself?